server/src/routes/api.rs

```diff
@@ -40,10 +40,15 @@ pub fn config(cfg: &mut web::ServiceConfig, rate_limit: &RateLimit) {
           .route(web::get().to(route_get::<Search>)),
       )
       // Community
+      .service(
+        web::resource("/community")
+          .guard(guard::Post())
+          .wrap(rate_limit.post())
```

Comment:
I apologize, I should've been more clear, the community creation rate should use rate_limit_register, IE default 3 per hour.